static/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
The style guide states to use a lowercase doctype: http://contribute.jquery.org/style-guide/html/#doctype